src/Apps/Auction/__tests__/redirects.test.ts

```diff
@@ -25,23 +25,23 @@ describe("Auction/redirects", () => {
   }
 
   const mockResolver = (
-    data: RegisterQueryResponse
-  ): RegisterQueryResponse => ({
+    data: DeFragedRegisterQueryResponse
+  ): DeFragedRegisterQueryResponse => ({
     sale: data.sale,
     me: data.me,
   })
 
   it("does not redirect if a sale is found", async () => {
-    const { redirect } = await render(
+    const result = await render(
       `/auction-registration2/${Fixture.sale.id}`,
       mockResolver(Fixture)
     )
-
+    const { redirect } = result
     expect(redirect).toBeUndefined
   })
 
   it("redirects to the auction registration modal if the user has a qualified credit card", async () => {
-    const { redirect } = await render(
+    const result = await render(
```

Comment:
This one too. It's not evident why the destructuring was removed. Was it for debugging purposes?

Reply:
correct.